model/src/main/java/org/cbioportal/model/summary/MutationSummary.java

```diff
@@ -0,0 +1,259 @@
+package org.cbioportal.model.summary;
+
+import java.io.Serializable;
+
+public abstract class MutationSummary implements Serializable {
```

Comment:
is the idea to move some of the logic from the services to this model objects eventually? In their current form they are pretty [anemic](http://www.martinfowler.com/bliki/AnemicDomainModel.html)